### PR TITLE
Add pages that list all missions/events

### PR DESCRIPTION
**Pull request by mcosand**
Add a simple view that includes all activities (missions or events). Intended to be low-cost initial implementation. Does not add links from existing UI into these pages, the user needs to type the page address (https://respond.team.org/mission or https://respond.team.org/event) into their browser.

### CODE REVIEW

**Comment by mcosand** [comment]
[Overflow page for all activities](https://trello.com/c/yFS2VPLL/23-overflow-page-for-all-activities)

File: src/app/(main)/EventListPage.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    <OutputText label="Location" value={a.location.title} />
+                    <OutputText label="State #" value={a.idNumber} />
+                  </Box>
+                  <Box>
+                    <OutputText label="Mission Status" value={getActivityStatus(a)} />
+                    {isFuture(a.startTime) && <OutputTime label="Start Time" time={a.startTime}></OutputTime>}
+                    <OutputText label="Active Participants" value={getActiveParticipants(a).length.toString()} />
+                  </Box>
+                </OutputForm>
+              </EventTile>
```
Comment (colincwilliams-kcsara) [inline src/app/(main)/EventListPage.tsx:44]: I'm not against this duplication going in to get these pages lit up, but can you file a bug in Trello to track de-duping the display of activity tiles?

Comment (mcosand) [reply src/app/(main)/EventListPage.tsx:44]: taking another stab at this. Might sneak in some spacing between Missions and Events on the main page.

File: src/app/(main)/EventListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { EventTile } from './EventTile';
+import { OutputForm, OutputText, OutputTime } from '@respond/components/OutputForm';
+import { apiFetch } from '@respond/lib/api';
+
+export const EventListPage = ({activityType }: { activityType: ActivityType }) => {
+  const [ loading, setLoading ] = React.useState<boolean>(true);
+  const [ activities, setActivities ] = React.useState<Activity[]>([]);
+
+  React.useEffect(() => {
+    apiFetch<{ data: Activity[] }>(`/api/v1/${activityType}`).then(api => {
```
[the remark this answers is not in the document]
Comment (colincwilliams-kcsara) [reply src/app/(main)/EventListPage.tsx:15]: That's fair, and I can get behind the split of "real-time" vs "not real-time" data. Ok, I'm on board with it!

**Pull request closed without merging**